project/build.properties

```diff
@@ -14,4 +14,4 @@
 # the License.
 #
 
-sbt.version=0.13.2
\ No newline at end of file
+sbt.version=0.13.9
```

Comment:
Not related to this change

Reply:
Actually it is :) dependency resolution error due to spark 2.1 is fixed by sbt 0.13.11